eva-accession-import/src/test/java/uk/ac/ebi/eva/accession/dbsnp/model/DbsnpVariantAllelesTest.java

```diff
@@ -165,6 +165,40 @@ public void reverseSTRAlleles() {
         assertEquals(Arrays.asList("GAGAGAGAGAGAGAGATT", "TATATA"), reverseAllelesSTR.getAllelesInForwardStrand());
     }
 
+    @Test
+    public void dashCountSTRAlleles() {
+        DbsnpVariantAlleles dashCountOnly = new DbsnpVariantAlleles("AT", "(AT)-", Orientation.FORWARD,
+                                                                    Orientation.FORWARD,
+                                                                    DbsnpVariantType.MICROSATELLITE);
+
+        assertEquals("AT", dashCountOnly.getReferenceInForwardStrand());
+        assertEquals(Arrays.asList(""), dashCountOnly.getAllelesInForwardStrand());
+
+        DbsnpVariantAlleles dashAndOtherCounts = new DbsnpVariantAlleles("AC", "(AC)3/(A)1(AC)2/(A)1(AC)1/(A)1(AC)-",
+                                                                         Orientation.FORWARD, Orientation.FORWARD,
+                                                                         DbsnpVariantType.MICROSATELLITE);
+
+        assertEquals("AC", dashAndOtherCounts.getReferenceInForwardStrand());
+        assertEquals(Arrays.asList("ACACAC", "AACAC", "AAC", "A"), dashAndOtherCounts.getAllelesInForwardStrand());
+    }
+
+    @Test
+    public void noCountSTRAlleles() {
+        DbsnpVariantAlleles emptyCountOnly = new DbsnpVariantAlleles("AT", "(AT)", Orientation.FORWARD,
+                                                                     Orientation.FORWARD,
+                                                                     DbsnpVariantType.MICROSATELLITE);
+
+        assertEquals("AT", emptyCountOnly.getReferenceInForwardStrand());
+        assertEquals(Arrays.asList(""), emptyCountOnly.getAllelesInForwardStrand());
```

Comment:
Shouldn't this be "AT" though? If unaccompanied by any number, shouldn't the bases be expanded as it is?

Reply:
Oops, I think you are right on this one.